### ISSUE TITLE
Elementor 2.5.1 Video Widget Bug

### ISSUE THREAD

**Issue by nerdtech** [elementor/elementor#7311]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

The video widget disappears when you apply "fixed" or "absolute" positioning.

## Steps to reproduce

1. Drag elementors video widget into a section.
2. Edit the widget and go to the "advanced" tab.
3. Select "custom positioning".
4. Under positioning apply either "fixed" or "absolute".
5. The widget disappears.

## Isolating the problem

- [x ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x ] This bug happens with a default WordPress theme active.
- [ x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>
					
== Server Environment ==
	Operating System: WINNT
	Software: Apache
	MySQL version: MySQL Community Server - GPL v8.0.15
	PHP Version: 7.2.15
	PHP Max Input Vars: 1000
	PHP Max Post Size: 256M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.1
	Site URL: http://localhost/wordpress
	Home URL: http://localhost/wordpress
	WP Multisite: No
	Max Upload Size: 256 MB
	Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Elementor Hello Theme
	Version: 1.1.0
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.119 Safari/537.36

== Active Plugins ==
	Advanced Custom Fields PRO
		Version: 5.7.12
		Author: Elliot Condon

	All-in-One WP Migration
		Version: 6.77
		Author: ServMask

	Classic Editor
		Version: 1.4
		Author: WordPress Contributors

	Columns Alignment Fix for Elementor
		Version: 1.1.0
		Author: SukiWP

	Custom Post Type UI
		Version: 1.6.1
		Author: WebDevStudios

	Elegant Tabs for Elementor
		Version: 1.0
		Author: InfiWebs

	Elementor
		Version: 2.5.1
		Author: Elementor.com

	Elementor Pro
		Version: 2.4.6
		Author: Elementor.com

	JetSmartFilters
		Version: 1.2.1
		Author: Zemez

	reSmush.it Image Optimizer
		Version: 0.1.22
		Author: reSmush.it

	WP Nag Hide
		Version: 1.0
		Author: Milton Keynes Web Design

	WP Rocket
		Version: 3.2.5
		Author: WP Media

== Log ==
	: 
JS: showing 20 of 36JS: 2019-02-21 09:08:41 [error X 1][https://nerdtech.uk/wp-admin/post.php?post=802&action=elementor:2:1] onelementor is not defined 
JS: 2019-02-21 09:08:41 [error X 1][https://nerdtech.uk/wp-admin/post.php?post=802&action=elementor:2:1] oncelementor is not defined 
JS: 2019-02-21 09:08:41 [error X 1][https://nerdtech.uk/wp-admin/post.php?post=802&action=elementor:2:1] onclielementor is not defined 
JS: 2019-02-21 09:08:42 [error X 1][https://nerdtech.uk/wp-admin/post.php?post=802&action=elementor:2:1] onclickelementor is not defined 
JS: 2019-02-21 09:09:08 [error X 1][https://nerdtech.uk/wp-admin/post.php?post=802&action=elementor:2:9] Invalid left-hand side in assignment 
JS: 2019-02-21 09:14:24 [error X 1][https://nerdtech.uk/wp-admin/post.php?post=802&action=elementor:2:10] Unexpected token ( 
JS: 2019-02-21 09:19:58 [error X 4][https://nerdtech.uk/wp-admin/post.php?post=802&action=elementor:2:122] Unexpected token ) 
JS: 2019-02-21 09:21:58 [error X 10][https://nerdtech.uk/wp-admin/post.php?post=802&action=elementor:2:1] $ is not a function 
JS: 2019-02-21 09:22:02 [error X 1][https://nerdtech.uk/wp-admin/post.php?post=802&action=elementor:2:1] jq is not defined 
JS: 2019-02-21 09:22:03 [error X 1][https://nerdtech.uk/wp-admin/post.php?post=802&action=elementor:2:1] jque is not defined 
JS: 2019-02-21 09:22:03 [error X 1][https://nerdtech.uk/wp-admin/post.php?post=802&action=elementor:2:1] jquery is not defined 
JS: 2019-02-21 09:22:07 [error X 1][https://nerdtech.uk/wp-admin/post.php?post=802&action=elementor:2:1] juery is not defined 
JS: 2019-02-21 09:22:47 [error X 2][https://nerdtech.uk/wp-admin/post.php?post=802&action=elementor:2:1] jQ is not defined 
JS: 2019-02-21 09:22:47 [error X 1][https://nerdtech.uk/wp-admin/post.php?post=802&action=elementor:2:1] jQu is not defined 
JS: 2019-02-21 11:12:43 [error X 16][https://nerdtech.uk/wp-admin/post.php?post=802&action=elementor:1:1] Unexpected token < 
JS: 2019-02-22 09:09:23 [error X 3][https://nerdtech.uk/wp-content/plugins/elementor/assets/lib/nouislider/nouislider.min.js?ver=13.0.0:2:22846] noUiSlider (13.0.0): Slider was already initialized. 
JS: 2019-03-03 16:40:40 [error X 1][http://localhost/wordpress/wp-admin/post.php?post=802&action=elementor:2:1] j is not defined 
JS: 2019-03-03 16:40:42 [error X 1][http://localhost/wordpress/wp-admin/post.php?post=802&action=elementor:2:1] jQuer is not defined 
JS: 2019-03-04 12:28:13 [error X 1][http://localhost/wordpress/wp-content/plugins/elementor/assets/js/editor.min.js?ver=2.4.7:2:206234] noUiSlider is not defined 
JS: 2019-03-04 12:39:41 [error X 14][http://localhost/wordpress/wp-content/plugins/elementor/assets/js/frontend.min.js:2:41754] Cannot read property \'indexOf\' of undefined 

Log: showing 7 of 72019-02-18 20:31:59 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.4.6',
  'to' => '2.4.7',
)]
2019-02-19 20:59:52 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.4.4',
  'to' => '2.4.5',
)]
2019-02-20 21:50:36 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.4.7',
  'to' => '2.5.0-beta1',
)]
2019-02-21 15:23:44 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.5.0-beta1',
  'to' => '2.5.0-beta2',
)]
2019-03-04 12:28:34 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.5.0-beta2',
  'to' => '2.5.0',
)]
2019-03-04 16:56:55 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '2.5.0',
  'to' => '2.5.1',
)]
2019-03-05 07:45:40 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.4.5',
  'to' => '2.4.6',
)]
</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by shilo-ey** [comment]
Hi @nerdtech 

Thanks for reporting.

We will take a look and update this issue.

thanks

**Comment by arielk** [comment]
Fix for v2.5.2

Thanks for your feedback
